py/strato/common/log/log2text.py

```diff
@@ -56,25 +65,62 @@ def __init__(self, relativeTime, withThreads, showFullPaths, noDebug, microsecon
+
+    def _getLogTypeConf(self, logPath):
+        for logType in self.configFile['logTypes']:
+            for pattern in logType['paths']:
+                if re.compile(pattern).match(os.path.basename(logPath)):
+                    return logType
+        return None
+
+    def _processGenericLog(self, line, logConf):
+        try:
+            msg, timestamp = lineparse.seperateTimestamp(line, logConf['logFormat']['timestamp'])
+            epochTime = lineparse.translateToEpoch(timestamp, logConf['timeStampFormat'])
+            if logConf['timezoneOffset'] == 'localtime':
+                epochTime += self._localTimezoneOffset
+            return line.strip().replace(timestamp, self._clock(epochTime)), epochTime
+        except:
+            # in case the line wasn't able to get parsed for some reason, print it as when you encounter it
+            return line.strip('\n'), HIGHEST_PRIORITY
+
+    def _processStratolog(self, line):
+        parsedLine = json.loads(line)
+        if parsedLine['levelno'] < self._minimumLevel:
             return None
-        if 'args' in obj and obj['args']:
-            if isinstance(obj['args'], (dict, tuple)):
-                message = obj['msg'] % obj['args']
-            elif isinstance(obj['args'], list):
-                message = obj['msg'] % tuple(obj['args'])
+
+        if 'args' in parsedLine and parsedLine['args']:
+            if isinstance(parsedLine['args'], (dict, tuple)):
+                message = parsedLine['msg'] % parsedLine['args']
+            elif isinstance(parsedLine['args'], list):
+                message = parsedLine['msg'] % tuple(parsedLine['args'])
             else:
-                message = obj['msg']
+                message = parsedLine['msg'].replace('%', '%%')
         else:
-            message = obj['msg'].replace('%', '%%')
-        clock = self._clock(obj['created'])
-        colorPrefix = self._COLORS.get(obj['levelno'], '')
+            message = parsedLine['msg'].replace('%', '%%')
+        clock = self._clock(parsedLine['created'])
+        colorPrefix = self._COLORS.get(parsedLine['levelno'], '')
         formatted = self._logFormat % dict(
-            obj, message=message, log2text_clock=clock, log2text_colorPrefix=colorPrefix)
-        if obj['exc_text']:
-            formatted += "\n" + obj['exc_text']
-        return formatted
+            parsedLine, message=message, log2text_clock=clock, log2text_colorPrefix=colorPrefix)
+        if parsedLine['exc_text']:
+            formatted += "\n" + parsedLine['exc_text']
+        return formatted, parsedLine['created']
+
+    def _processExceptionLog(self, line):
+        parsedLine = json.loads(line)
+        line = parsedLine['message']
+        logPath = parsedLine['source']
+        if logPath not in self._exceptionFiles:
+            self._exceptionFiles[logPath] = _getColorCode(len(self._exceptionFiles))
+        logTypeConf = self._getLogTypeConf(logPath)
```

Comment:
why sometimes you called it logTypeConf and sometimes logConf? Please be consistent with the names.
And again, no need to call it multiple times